lib/client/terminal/terminal_windows.go

```diff
@@ -0,0 +1,279 @@
+//go:build windows
+// +build windows
```

Comment:
Shouldn't need the +build line anymore, right?

Reply:
In fact, you shouldn't need any build tags at all since the file is named terminal_windows.go

Reply:
You're right, we can remove them at least from `terminal_windows.go`. I'm not sure we've fully moved to Go 1.17 everywhere, though, so it might be best to keep `+build` in `terminal_unix.go` for now?

Reply:
Gotcha, makes sense 👍